From ffdb09686408dd5faea129b7310d3fc72de186a8 Mon Sep 17 00:00:00 2001
From: dbw9580 <dbw9580@live.com>
Date: Tue, 15 Jan 2019 21:12:32 +0800
Subject: [PATCH] fix sdk version

---
 lib/build.gradle | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/lib/build.gradle b/lib/build.gradle
index 2260331..eab8981 100644
--- a/lib/build.gradle
+++ b/lib/build.gradle
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.2"
+    compileSdkVersion 26
+    buildToolsVersion "26.0.2"
     
     defaultConfig {
         minSdkVersion 21
-        targetSdkVersion 25
+        targetSdkVersion 26
         versionCode 1
         versionName "1.0"
     }
-- 
2.20.1

